yarn.lock

```diff
@@ -20447,7 +20431,6 @@ pkg-dir@^4.1.0, pkg-dir@^4.2.0:
 
 pkg-fetch@3.4.2, "pkg-fetch@https://github.com/aws-amplify/pkg-fetch#ad4a21feb533d338bf951e7ba28cea7256aedeff":
   version "3.4.2"
-  uid ad4a21feb533d338bf951e7ba28cea7256aedeff
```

Comment:
this needs to stay here otherwise we will have security issues with dependabot

Reply:
Is there anything I need to do to my machine to trigger this to not be removed, or are we manually managing the yarn.lock for this for now?